CIP-0019/README.md

```diff
@@ -12,19 +12,43 @@ License: CC-BY-4.0
 
 # Abstract
 
-This specification describes the binary structure of addresses in Cardano, covering both addresses introduced in the Shelley era and the legacy format from the Byron era.
+This specification describes the structure of addresses in Cardano, covering both addresses introduced in the Shelley era and the legacy format from the Byron era.
 
 # Motivation
 
 Document design choices for posterity. Most applications interacting with the Cardano blockchain will likely not have any need for this level of details, however, some might. This CIP is meant to capture this knowledge. 
 
 # Specification
 
-## Overview
+## Introduction
 
-In Cardano, addresses are comprised of two parts, a **header** and a **payload**. Depending on the header, the interpretation of the payload varies. 
+In Cardano, an address is a **sequence of bytes** that conforms to a particular format, which we describe below.
 
-In the header, bits [7;4] indicate the type of addresses being used; we'll call these four bits the **header type**. The remaining four bits [3;0] are either unused or refer to what we'll call the **network tag**. There are currently 11 types of addresses in Cardano which we'll divide into three categories: [Shelley addresses], [stake addresses], and [Byron addresses]. 
+However, users will typically come into contact with addresses only after these addresses have been **encoded** into sequences of human-readable characters. In Cardano, the [Bech32][] and [Base58][] encodings are used to encode addresses, as opposed to standard hexadecimal notation (Base16, example `0x8A7B`). These encoded sequence of characters have to be distinguished from the byte sequences that they encode, but lay users will (and should) perceive the encoded form as "the" address.
```

Comment:
[Bech32]
[Base58]

[Bech32]: https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki
[Base58]: https://tools.ietf.org/id/draft-msporny-base58-01.html

Reply:
Yes? Markdown resolves the strings [Bech32][] and [Base58][] ("implicit link name") to [Bech32][Bech32] and [Base58][Base58] ("reference-style links") which are then hyperlinked using the link destinations at the end of the text. I have acquired the habit of adding [] at the end, because [Bech32] without the empty set of square brackets [] is not a link according to the initial [Daring Fireball][markdown] specification.

  [markdown]: https://daringfireball.net/projects/markdown/syntax